.github/workflows/CI.yml

```diff
@@ -80,21 +80,13 @@ jobs:
     uses: ./.github/workflows/prettier.yml
 
   validate_local_links:
-    name: Validate Markdown
-    needs: [setup, prettier]
+    name: Validate
+    needs: [setup]
```

Comment:
I figured there wasn't much need for this step to wait for Prettier, but curious to hear your thoughts on this?

Reply:
I think we can run all the non formatting stuff (basically everything except eslint and jsonlint right now) without formatting if we want. I originally included it so all none of the subsequent jobs would run if the formatting failed to save compute resources but it might be better to just let them run so you can see all non formatting errors at first glance as well.